Lib/fontbakery/checks/vendorspecific/googlefonts/conditions.py

```diff
@@ -298,17 +300,34 @@ def remote_styles(font):
             continue
         file_obj = download_file(dl_url)
         if file_obj:
-            remote_fonts.append([filename, TTFont(file_obj)])
+            remote_fonts.append(TTFont(file_obj))
 
     rstyles = {}
-    for remote_filename, remote_font in remote_fonts:
-        remote_style = os.path.splitext(remote_filename)[0]
-        if "-" in remote_style:
-            remote_style = remote_style.split("-")[1]
-        rstyles[remote_style] = remote_font
+    for font in remote_fonts:
+        if "fvar" in font:
+            for instance in font["fvar"].instances:
+                inst_subfamily = (
+                    font["name"]
+                    .getName(instance.subfamilyNameID, 3, 1, 0x409)
+                    .toUnicode()
+                )
+                rstyles[inst_subfamily] = font
+        else:
+            rstyles[font["name"].getBestSubFamilyName()] = font
     return rstyles
 
 
+@condition(Font)
+def remote_style(font):
+    font_style = font.ttFont["name"].getBestSubFamilyName()
+    remote_styles = font.remote_styles
+    if not remote_styles:
+        return None
+    if font_style in remote_styles:
+        return remote_styles[font.style]
```

Comment:
Should be `font_style`? (If so, consider using `remote_styles.get(font_style)` instead of if statement)

Reply:
Good catch. Fixed.